CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ Adding a new version? You'll need three changes:
 
 ### Breaking Changes
 
+- `ControlPlane` `v2alpha1` has been replaced by `ControlPlane` `v2beta1`
```

Comment:
I think that for now this is good but at the release time (and when we merge the entries of 2.0 and 2.0 alphas) we should get rid of mentions of `v2alpha1` and just use `v2beta1` throughout the changelog.

WDYT?

(specifically I'm thinking of places like https://github.com/Kong/kubernetes-configuration/blob/a073cd7cd0493b46d222bf2821d97653e3d71a87/CHANGELOG.md?plain=1#L100-L106)